runtime/js/30_os.js

```diff
@@ -24,13 +24,24 @@
     return core.jsonOpSync("op_system_cpu_info");
   }
 
+  let exitHandler = null;
+  function setExitHandler(fn) {
+    exitHandler = fn;
+  }
```

Comment:
Please add a comment explaining the purpose of this handler along with info that it's not a public API and can be used only by internal test runner